Cargo.toml

```diff
@@ -11,7 +11,7 @@ license = "MPL-2.0"
 [dependencies]
 dusk-bytes = "0.1"
 dusk-poseidon = { version = "0.30", default-features = false, features = ["rkyv-impl", "alloc"] }
-poseidon-merkle = { version = "0.2.1-rc.0", features = ["rkyv-impl", "zk", "size_32"] }
+poseidon-merkle = { version = "0.2", features = ["rkyv-impl", "zk", "size_32"] }
```

Comment:
Are you donwgrading this on purpose?

Reply:
yes, it has been released!

Reply:
Resolving, cause tests are passing with 0.2.0 too